int64 waypointTotal
int64 waypointEnCours